test/integration/PermissionTable.test.ts

```diff
@@ -0,0 +1,218 @@
+
+  [ 'PUT',     'C/',  [],                     undefined,              '',     N3,  401, 401 ],
+  [ 'PUT',     'C/',  [ AM.read ],            undefined,              '',     N3,  401, 401 ],
+  [ 'PUT',     'C/',  [ AM.write ],           undefined,              '',     N3,  205, 201 ],
+
+  [ 'PUT',     'C/R', [],                     undefined,              '',     TXT, 401, 401 ],
+  [ 'PUT',     'C/R', [],                     [ AM.read ],            '',     TXT, 401, 401 ],
+  [ 'PUT',     'C/R', [],                     [ AM.append ],          '',     TXT, 401, 401 ],
+  [ 'PUT',     'C/R', [],                     [ AM.write ],           '',     TXT, 205, 401 ],
+  [ 'PUT',     'C/R', [ AM.read ],            undefined,              '',     TXT, 401, 401 ],
+  [ 'PUT',     'C/R', [ AM.append ],          undefined,              '',     TXT, 401, 401 ],
+  [ 'PUT',     'C/R', [ AM.write ],           undefined,              '',     TXT, 205, 201 ],
+  [ 'PUT',     'C/R', [ AM.append ],          [ AM.write ],           '',     TXT, 205, 201 ],
+
+  [ 'PATCH',  'C/R', [],                     undefined,              DELETE,  N3,  401, 401 ],
+  // We don't return 404 yet in case a PATCH has no inserts and C/R does not exist
+  // [ 'PATCH',   'C/R', [],                     [ AM.read ],            DELETE,  N3,  401, 404 ],
+  [ 'PATCH',   'C/R', [],                     [ AM.append ],          INSERT, N3,  205, 401 ],
+  [ 'PATCH',   'C/R', [],                     [ AM.append ],          DELETE, N3,  401, 401 ],
+  [ 'PATCH',   'C/R', [],                     [ AM.write ],           INSERT, N3,  205, 401 ],
+  [ 'PATCH',   'C/R', [],                     [ AM.write ],           DELETE, N3,  401, 401 ],
+  [ 'PATCH',   'C/R', [ AM.append ],          [ AM.write ],           INSERT, N3,  205, 201 ],
+  [ 'PATCH',   'C/R', [ AM.append ],          [ AM.write ],           DELETE, N3,  401, 401 ],
+  // We don't return 404 yet in case a PATCH has no inserts and C/R does not exist
+  // [ 'PATCH',   'C/R', [],                     [ AM.read, AM.write ],  DELETE, N3,  205, 404 ],
+
+  [ 'DELETE',  'C/R', [],                     undefined,              '',     '',  401, 401 ],
+  [ 'DELETE',  'C/R', [],                     [ AM.read ],            '',     '',  401, 404 ],
+  [ 'DELETE',  'C/R', [],                     [ AM.append ],          '',     '',  401, 401 ],
+  [ 'DELETE',  'C/R', [],                     [ AM.write ],           '',     '',  401, 401 ],
+  [ 'DELETE',  'C/R', [ AM.read ],            undefined,              '',     '',  401, 404 ],
+  [ 'DELETE',  'C/R', [ AM.append ],          undefined,              '',     '',  401, 401 ],
+  [ 'DELETE',  'C/R', [ AM.append ],          [ AM.read ],            '',     '',  401, 404 ],
+  // Not sure why 401 is suggested here instead of 404?
+  // [ 'DELETE',  'C/R', [ AM.write ],           undefined,              '',     '',  205, 401 ],
```

Comment:
Requires a spec issue?

Reply:
Disclosing whether a resource exists requires `Read` access on its container.

Reply:
Solution will be to also require `Read` permissions for DELETE requests that target a container.

Reply:
Actually, that is the answer to the comment below. This one has the same issue as the GET above that we require the parent permissions in a situation where we don't easily have access to them. So this one is probably going to stay 401 for a bit then. I'll look if there is a possible change that also exposes parent permissions to the authorizer to handle this situation.